audio/data.py

```diff
@@ -258,66 +262,86 @@ def __init__(self, items, path, config=AudioConfig(), **kwargs):
         if str(path) not in str(cd): 
             config.cache_dir = path / cd
         self.config = config
+        self.nchannels = 1 if self.config.downmix else None
         self.copy_new += ['config']
 
-    def open(self, item) -> AudioItem:
-        p = Path(item)
-        if self.path is not None and not p.exists() and str(self.path) not in str(item): p = self.path/item
-        if not p.exists(): raise FileNotFoundError(f"Neither '{item}' nor '{p}' could be found")
-        if not str(p).lower().endswith(AUDIO_EXTENSIONS): raise Exception("Invalid audio file")
```

Comment:
This code was written to make it easier to work with from_csv and from_df. Why did you remove it?